packages/html/src/spreadsheet/tests/spreadsheet-row-contextmenu.tsx

```diff
@@ -0,0 +1,538 @@
+                    <div className="k-list-content k-list-scroller" unselectable="on"></div>
+                </div>
+                <div className="k-spreadsheet-sheets-bar">
+                    <Button className="k-spreadsheet-sheet-add" fillMode="flat" icon="plus" />
+                    <Button className="k-spreadsheet-sheets-menu" fillMode="flat" icon="menu" />
+                    <div className="k-spreadsheet-sheets k-tabstrip k-tabstrip-bottom k-tabstrip-scrollable">
+                        <div className="k-tabstrip-items-wrapper k-vstack">
+                            <ul className="k-reset k-tabstrip-items">
+                                <li className="k-item k-first">
+                                    <span className="k-link">Sheet 1</span>
+                                    <MenuButton size="medium" showArrow={false} icon="caret-alt-down" fillMode="flat" />
+                                </li>
+                                <li className="k-item">
+                                    <span className="k-link">Sheet 2</span>
+                                    <MenuButton size="medium" showArrow={false} icon="caret-alt-down" fillMode="flat" />
+                                </li>
+                                <li className="k-item k-active">
+                                    <span className="k-link">Sheet 3</span>
+                                    <MenuButton size="medium" showArrow={false} icon="caret-alt-down" fillMode="flat" />
+                                </li>
+                                <li className="k-item k-focus">
+                                    <span className="k-link">Sheet 4</span>
+                                    <MenuButton size="medium" showArrow={false} icon="caret-alt-down" fillMode="flat" />
+                                </li>
+                                <li className="k-item">
+                                    <span className="k-link">Sheet 5</span>
+                                    <MenuButton size="medium" showArrow={false} icon="caret-alt-down" fillMode="flat" />
+                                </li>
+                                <li className="k-item k-last k-disabled">
+                                    <span className="k-link">Sheet 6</span>
+                                    <MenuButton size="medium" showArrow={false} icon="caret-alt-down" fillMode="flat" />
+                                </li>
+                            </ul>
+                            <Button className="k-tabstrip-prev" fillMode="flat" icon="caret-alt-left" rounded={null} size={null} />
+                            <Button className="k-tabstrip-next" fillMode="flat" icon="caret-alt-right" rounded={null} size={null} />
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <Popup id="spreadsheet-context-menu-row" className="k-menu-popup">
```

Comment:
Shouldn't we add _k-context-menu-popup_ class too?

Reply:
Added